item-spec/item-spec.md

```diff
@@ -298,7 +298,9 @@ For more information on how to use roles see the [Asset Roles](../best-practices
 As detailed above, Items contain properties, which are the main source of metadata for searching across Items. Many content
 extensions can add further property fields as well. Any property that can be specified for an Item can also be specified for
 a specific asset. This can be used to override a property defined in the Item, or to specify fields for which there is no
-single value for all assets.
+single value for all assets. If a property is defined at the STAC entity level (Item or Collection), it is the default value for all assets.
+The values in the top level properties are considered nominal for the the Item or the Collection and thus may be adjusted for
+specific assets (see example below).
```

Comment:
Trying to make it more clear, also this is just the Item Spec so Collections should have this paragraph, too.

```suggestion
single value for all assets.
If a property is defined in the Item Properties, it is the default value for all assets.
Assets may override the properties in the Item Properties for specific assets (see example below).
Note that in some cases the inheritance may not lead to the expected results if other semantics have been defined for the property.
See the examples for `gsd` and `eo:bands` below, but it also applies to some other fields such as `created` or `updated`.
```